cmd/upgrade/cluster/cmd.go

```diff
@@ -212,8 +212,7 @@ func createUpgradePolicyHypershift(r *rosa.Runtime, cmd *cobra.Command, clusterK
 	cluster *cmv1.Cluster, version string, scheduleDate string, scheduleTime string) error {
 	upgradePolicyBuilder := cmv1.NewControlPlaneUpgradePolicy().ScheduleType("manual").
 		UpgradeType("ControlPlane").Version(version)
-	// TODO: remove forceNow when we support scheduling
-	nextRun := buildUpgradeSchedule(r, cmd, scheduleDate, scheduleTime, true)
+	nextRun := buildUpgradeSchedule(r, cmd, scheduleDate, scheduleTime)
```

Comment:
Are these changes supposed to be included in this PR? Seems unrelated

Reply:
they aren't, just removed them